resources/features/refapp-3.x/02-patient-registration/patient-registration.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Patient Registration
+
+  Background:
+    Given the user login to the Outpatient Clinic
+
+  Scenario Outline: Register a patient
+    When the user clicks on the add patient icon
+    And the user enters "<validity>" details for Andria Faiza
+    And the user clicks on the create patient button
+    Then the patient registration should be "<status>"
+    Examples:
+      | validity   | status       |
+      | correct    | successful   |
+      | wrong      | unsuccessful |
```

Comment:
Lets emphasise adding one last line to cater for new changes. Otherwise great work. @pasindur99  @pasindur99